### PR TITLE
Moved logger from dapr/dapr

### PR DESCRIPTION
**Pull request by pkedy**
Direct copy from dapr/dapr/pkg/logger with the exception of removing the dependency on dapr/dapr/pkg/version. I replaced that with a variable that must be set called `DaprVersion`.

### CODE REVIEW

**Comment by pkedy** [comment]
Forgot the Makefile and github actions workflow.  Adding now.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+################################################################################
+# Target: check-diff                                                           #
+################################################################################
+.PHONY: check-diff
+check-diff:
+	git diff --exit-code ./go.mod # check no changes
+
+################################################################################
+# Target: conf-tests                                                           #
```
Comment (artursouza) [inline Makefile:81]: Remove this, please.

File: .github/workflows/kit.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+# Copyright (c) Microsoft Corporation and Dapr Contributors.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: kit
+
+on:
+  push:
+    branches:
+      - master
```
Comment (artursouza) [inline .github/workflows/kit.yml:11]: `master` branch is not called `main` - this probably explains why PR action did not get triggered.

File: logger/dapr_logger.go [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+		}),
+	}
+
+	dl.EnableJSONOutput(defaultJSONOutput)
+
+	return dl
+}
+
+// EnableJSONOutput enables JSON formatted output log
+func (l *daprLogger) EnableJSONOutput(enabled bool) {
```
Comment (artursouza) [inline logger/dapr_logger.go:40]: Please, get the version as param instead.

File: logger/dapr_logger.go [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+// daprLogger is the implemention for logrus
+type daprLogger struct {
+	// name is the name of logger that is published to log as a scope
+	name string
+	// loger is the instance of logrus logger
+	logger *logrus.Entry
+}
+
+var GetVersion = func() string {
```
Comment (artursouza) [inline logger/dapr_logger.go:18]: remove this global var and get version via param in the method where this is used. See comment below.

**Review by artursouza — CHANGES_REQUESTED** [review]

File: logger/dapr_logger.go (hunk 3 above)
Comment (pkedy) [reply logger/dapr_logger.go:40]: The problem with is that is called by `newDaprLogger` which is not exposed by the package. I propose keeping this as a global variable (change from func to a string variable) to keep the changes in dapr/dapr to a minimum. There is a much more elegant way of handling this but it is outside the scope of this PR.

Comment (artursouza) [reply logger/dapr_logger.go:38]: I see it. OK, we can refactor that later.

**Review by artursouza — APPROVED** [review]

**Pull request merged**